From 910b74fb0e0369b18aa689ab02c9413235c18f98 Mon Sep 17 00:00:00 2001
From: Gabor Juhos <juhosg@openwrt.org>
Date: Sun, 9 Dec 2012 16:47:35 +0100
Subject: [PATCH 2/4] ath9k: add EEPROM offset to debug message

Show the EEPROM offset of the failed read operation
in 'ath9k_hw_nvram_read'. The debug message is more
informative this way.

Signed-off-by: Gabor Juhos <juhosg@openwrt.org>
---
 drivers/net/wireless/ath/ath9k/eeprom.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/net/wireless/ath/ath9k/eeprom.c
+++ b/drivers/net/wireless/ath/ath9k/eeprom.c
@@ -119,7 +119,8 @@ bool ath9k_hw_nvram_read(struct ath_comm
 
 	ret = common->bus_ops->eeprom_read(common, off, data);
 	if (!ret)
-		ath_dbg(common, EEPROM, "Unable to read eeprom region\n");
+		ath_dbg(common, EEPROM,
+			"unable to read eeprom region at offset %u\n", off);
 
 	return ret;
 }
